methods/index.json

```
@@ -0,0 +1,1910 @@
+    "contactWebsite": "",
+    "specification": "https://github.com/w3c-ccg/did-method-web"
+  },
+  {
+    "name": "web7",
+    "status": "registered",
+    "verifiableDataRegistry": "Web 7.0 Foundation",
+    "contactName": "Web 7.0 Foundation",
+    "contactEmail": "mwherman@parallelspace.net",
+    "contactWebsite": "http://hyperonomy.com",
+    "specification": "https://github.com/Web7Foundation/Specifications/blob/main/methods/did-web7-1-0-1.md"
+  },
+  {
+    "name": "webs",
+    "status": "registered",
+    "specification": "https://trustoverip.github.io/tswg-did-method-webs-specification/",
+    "contactName": "Trust over IP (ToIP) DID Method webs Task Force",
+    "contactEmail": "technology-stack-wg@lists.trustoverip.org",
+    "contactWebsite": "https://wiki.trustoverip.org/display/HOME/DID+Method+webs+Task+Force",
+    "verifiableDataRegistry": "Web, and <a href=\"https://keri.one/\">Key Event Receipt Infrastructure (KERI)</a>"
+  },
+  {
+    "name": "webvh",
+    "status": "registered",
+    "verifiableDataRegistry": "Web",
+    "contactName": [
+      "Stephen Curran",
+      "John Jordan",
+      "Andrew Whitehead",
+      "Brian Richter",
+      "Michel Sahli",
+      "Martina Kolpondinos",
+      "Dmitri Zagidulin"
+    ],
+    "contactEmail": [
+      "swcurran@cloudcompass.ca",
+      "martina@kosmaconnect.net",
+      "brian@aviary.tech"
+    ],
+    "contactWebsite": "https://didwebvh.info",
+    "specification": "https://identity.foundation/didwebvh/"
+  },
+  {
+    "name": "wlk",
+    "status": "registered",
+    "verifiableDataRegistry": "Weelink Network",
+    "contactName": "Weelink",
+    "contactEmail": "",
+    "contactWebsite": "",
+    "specification": "https://weelink-team.github.io/weelink/DIDDesignEn"
+  },
+  {
+    "name": "work",
+    "status": "registered",
+    "verifiableDataRegistry": "Hyperledger Fabric",
+    "contactName": "Workday, Inc.",
+    "contactEmail": "",
+    "contactWebsite": "",
+    "specification": "https://workday.github.io/work-did-method-spec/"
+  },
+  {
+    "name": "yourd",
+    "status": "registered",
+    "verifiableDataRegistry": "YourD compatible chains",
+    "contactName": "YourD",
+    "contactEmail": "",
+    "contactWebsite": "https://www.yourd.xyz/",
+    "specification": "https://github.com/DataSovereignty-YourD/YourD-did-sepcification"
+  },
+  {
+    "name": "zk",
+    "status": "registered",
+    "verifiableDataRegistry": "Arweave",
+    "contactName": "zCloak Network Research",
+    "contactEmail": "info@zcloak.network",
+    "contactWebsite": "https://zcloak.network/#/",
+    "specification": "https://github.com/zCloak-Network/zk-did-method-specs"
+  },
+  {
+    "name": "zkme",
+    "status": "registered",
+    "verifiableDataRegistry": "EVM compatible chains. Primary on Polygon",
+    "contactName": "zkMe",
+    "contactEmail": "",
+    "contactWebsite": "",
+    "specification": "https://github.com/zkMeLabs/zkme-did-method-spec"
+  }
+]
```

A minimallly visible commit, to re-trigger CI.
```suggestion
]

```